tests/client_test.go

```diff
@@ -453,6 +453,75 @@ func TestStats(t *testing.T) {
 	}
 }
 
+func TestStreamStats(t *testing.T) {
+	httpClient := &http.Client{}
+	c, err := client.NewNginxClient(httpClient, "http://127.0.0.1:8080/api")
+	if err != nil {
+		t.Fatalf("Error connecting to nginx: %v", err)
+	}
+
+	// need upstream for stats
+	server := client.StreamUpstreamServer{
+		Server: "127.0.0.1:8080",
+	}
+	err = c.AddStreamServer(streamUpstream, server)
+	if err != nil {
+		t.Errorf("Error adding stream upstream server: %v", err)
+	}
+
+	// make request so we have stream server zone stats, expect 5xx error
```

Comment:
since we're dealing with stream module here, to make things less confusing, try to establish a TCP connection using "net" package instead of using the httpclient. See https://golang.org/pkg/net/

Change the comment to something like below:
```
// Establish a connection to the stream server, so that we have some stats available for the stream zone.
```
Do not expect an error. If you want to have a simple stream server that responds something, see http://nginx.org/en/docs/stream/ngx_stream_return_module.html#return